src/main/java/planit/Planit.java

```diff
@@ -0,0 +1,39 @@
+package planit;
+
+import planit.messages.PlanitMessages;
+import planit.task.TaskManager;
+import planit.util.Echo;
+import planit.util.Ui;
+
+/**
+ * Represents the main entry point of Planit application.
+ */
+public class Planit {
+    /**
+     * Gets input from user regarding available choices.
+     *
+     * @param args Command-line arguments.
+     */
+    public static void main(String[] args) {
+        Ui.showPlanitWelcomeMessage();
+
+        TaskManager taskManager = new TaskManager();
+
+        boolean exitPlanit = false;
+        while (!exitPlanit) {
+            Ui.showPlanitUserGuide();
+            String userChoice = Ui.getUserInput();
+            if (userChoice.equalsIgnoreCase("planit") || userChoice.equals("1")) {
+                taskManager.run();
+            } else if (userChoice.equalsIgnoreCase("play") || userChoice.equals("2")) {
+                Echo.echo();
+            } else if (userChoice.equalsIgnoreCase("bye") || userChoice.equals("3")) {
+                exitPlanit = true;
+            } else {
+                Ui.showError(PlanitMessages.PLANIT_INVALID_OPTION);
```

Comment:
in your tP we would like to see one-shot commands rather than menu-based commands like this